### PR TITLE
a coat of pizzaz

### PR DESCRIPTION
**Pull request by potch**
What do you think?

screen shot 2014-09-15 at 10 32 40 pm

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,20 +2,48 @@
...
     <style>
+      html, body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+        overflow: hidden;
+      }
+      path {
+        stroke: rgba(255,255,255,.2) !important;
+        /*stroke-width: 0 !important;*/
```
Comment (cvan) [inline index.html]: remove?

File: index.html [hunk 2]
```diff
@@ -2,20 +2,48 @@
...
-        margin: 0 auto;
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
       }
       h1 {
+        font-family: 'Fira Sans', sans-serif;
+        font-size: 8vmin;
```
Comment (cvan) [inline index.html]: combine the two with a `font`

File: index.html [hunk 3]
```diff
@@ -2,20 +2,48 @@
...
+        background: linear-gradient(to bottom, #0095DD, #00539F);
+      }
       #container {
-        position: relative;
-        margin: 0 auto;
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
```
Comment (cvan) [inline index.html:21]: :fries:

File: index.html [hunk 4]
```diff
@@ -2,20 +2,48 @@
...
+    <link rel="stylesheet" href="http://code.cdn.mozilla.net/fonts/fira.css">
     <style>
+      html, body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+        overflow: hidden;
+      }
+      path {
+        stroke: rgba(255,255,255,.2) !important;
```
Comment (cvan) [inline index.html]: does datamaps try to override this?

Comment (potch) [inline index.html]: Yes, and the corresponding config to fix it wasn't working.

File: index.html (hunk 3 above)
Comment (potch) [inline index.html:21]: :eggplant:

**Comment by cvan** [comment]
looks very slick!

**Comment by nickdesaulniers** [comment]
Thanks for the coat of :pizza: :pizza: :pizza: !  This looks great!  Thanks @potch & @cvan !

**Comment by potch** [comment]
I found a fix to one or two more things, one sec!

**Comment by nickdesaulniers** [comment]
woah, finger hovering over the confirm merge button.  That was close!

File: my.js [hunk 5]
```diff
@@ -13,8 +10,12 @@ var close = '</strong></div>';
 var map = new Datamap({
   element: container,
   fills: {
-    SOLD: 'darkorange',
-    defaultFill: '#008EAB',
+    SOLD: '#FF9500',
+    defaultFill: 'rgba(234, 239, 242, 0.5)',
```
Comment (cvan) [inline my.js:14]: `0.5` -> `.5`

File: my.js [hunk 6]
```diff
@@ -1,6 +1,3 @@
-var container = document.getElementById('container');
-container.style.width = (window.innerWidth - 20) + 'px';
```
Comment (cvan) [inline my.js:2]: @potch how does the container get set now? and how does resizing work?

Comment (nickdesaulniers) [inline my.js:2]: inline css, see style tag above

Comment (potch) [inline my.js:2]: tied to the screen size, as opposed to inset

File: my.js (hunk 5 above)
Comment (potch) [inline my.js:14]: :eggplant:

**Comment by potch** [comment]
go for it!

File: my.js (hunk 5 above)
Comment (nickdesaulniers) [inline my.js:14]: What does the eggplant mean?

File: my.js (hunk 6 above)
Comment (cvan) [inline my.js:2]: aha, nice

File: my.js (hunk 5 above)
Comment (cvan) [inline my.js:14]: basically this:

Comment (potch) [inline my.js:14]: I use it to mean nitpick.

Comment (nickdesaulniers) [inline my.js:14]: I think @cvan thinks it means something else :P

**Pull request merged**